tests/integ/sagemaker/jumpstart/retrieve_uri/training.py

```diff
@@ -39,22 +40,20 @@ def __init__(
         hyperparameters,
         instance_type,
         training_dataset_s3_key,
-        suffix=None,
-        region=None,
-        boto_config=None,
-        base_name=None,
-        execution_role=None,
+        suffix=time.strftime("%Y-%m-%d-%H-%M-%S", time.gmtime()),
+        region=JUMPSTART_DEFAULT_REGION_NAME,
+        boto_config=Config(retries={"max_attempts": 10, "mode": "standard"}),
+        base_name="jumpstart-training-job",
+        execution_role=Session().get_caller_identity_arn(),
```

Comment:
Since this makes an http request, I think it is safer to keep it in the function body as you had it.